torch/testing/_internal/common_methods_invocations.py

```diff
@@ -10149,8 +10016,6 @@ def ref_pairwise_distance(input1, input2):
                         DecorateInfo(unittest.expectedFailure, 'TestMathBits', 'test_conj_view'),
                         DecorateInfo(unittest.expectedFailure, 'TestMathBits', 'test_neg_view'),
                         DecorateInfo(unittest.expectedFailure, 'TestMathBits', 'test_neg_conj_view'),
-                        # FIXME: ldexp does not accept scalar inputs
-                        DecorateInfo(unittest.expectedFailure, 'TestBinaryUfuncs', 'test_broadcast_python_scalar'),
```

Comment:
Why is this no longer needed? Are we excluding all scalars by setting `supports_rhs_python_scalar=False`?

Reply:
Yep.